script/Arcueid/Day 9/09_00_ARC09_3_1.txt

```diff
@@ -139,7 +139,7 @@ I guess the foundations of her personality were already in place, even back then
 -- Page 108, Offset 21573.
 -- 親父の手前、猫を五、六匹かぶっていたのかもしれない。
 She must have hidden her true self under five,
- maybe even six layers of disguises in front of our father.
+ or maybe even six layers of disguises in front of our father.
```

Comment:
Prefer without `or`

Reply:
yeah it felt good in the moment but i'll revert